http-client/src/main/java/io/airlift/http/client/HttpClientConfig.java

```diff
@@ -102,6 +102,20 @@
     private DataSize logBufferSize = new DataSize(1, MEGABYTE);
     private Duration logFlushInterval = new Duration(10, SECONDS);
     private boolean logCompressionEnabled = true;
+    private boolean hostnameVerificationEnabled = true;
```

Comment:
We could shorten to `verifyHostname`

Then the getter becomes `isVerifyHostname()`